src/vm/runners/cairo_runner.rs

```diff
@@ -2959,18 +2984,47 @@ mod tests {
     fn end_run() {
         let program = program!();
 
+        let hint_processor = BuiltinHintProcessor::new_empty();
         let mut cairo_runner = cairo_runner!(program);
         let mut vm = vm!();
 
         vm.accessed_addresses = Some(Vec::new());
-        assert_eq!(cairo_runner.end_run(true, false, &mut vm), Ok(()));
+        assert_eq!(
+            cairo_runner.end_run(true, false, &mut vm, &hint_processor),
+            Ok(()),
+        );
 
         cairo_runner.run_ended = false;
         cairo_runner.relocated_memory.clear();
-        assert_eq!(cairo_runner.end_run(true, true, &mut vm), Ok(()));
+        assert_eq!(
+            cairo_runner.end_run(true, true, &mut vm, &hint_processor),
+            Ok(()),
+        );
         assert!(!cairo_runner.run_ended);
     }
 
+    #[test]
+    fn end_run_proof_mode_insufficient_allocated_cells() {
+        let program = Program::from_file(
+            Path::new("cairo_programs/proof_programs/fibonacci.json"),
+            "main",
+        )
+        .unwrap();
```

Comment:
If you can, change this `unwrap` for `expect`

Reply:
Done.